### PR TITLE
Sync the updater.sh with the one in ToolForge

### PR DESCRIPTION
**Pull request by Ladsgroup**
- Installing npm locally and use its binary instead
 - Instead of dropping everything by deleting node_modules which would
   make the build really long (sometimes an hour or two), update it
   instead.
 - Do git reset --hard ot make sure the git repo is clean

### CODE REVIEW

File: build/updater.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-cd query-builder
+cd /data/project/query-builder-test/query-builder
+git reset --hard origin/master
 git pull
-rm -rf node_modules
-npm install -g npm@6.14.8
-npm install
-npm run build
+./node_modules/.bin/npm install npm@6.14.8
+./node_modules/.bin/npm update
```
Comment (jakobw) [inline build/updater.sh]: [`npm update`](https://docs.npmjs.com/cli/update) is for updating packages. We want `npm install` or `npm ci`.

File: build/updater.sh [hunk 2]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
-cd query-builder
+cd /data/project/query-builder-test/query-builder
+git reset --hard origin/master
 git pull
-rm -rf node_modules
-npm install -g npm@6.14.8
-npm install
-npm run build
+./node_modules/.bin/npm install npm@6.14.8
```
Comment (jakobw) [inline build/updater.sh]: This cannot work the first time it runs, can it? There won't be an npm binary inside `node_modules/` so this should just use the system's npm for this step, right?!

**Review by jakobw — CHANGES_REQUESTED** [review]
1 - 2 hours sounds really weird to me. We have setups for larger projects with the same tech stack and it takes a minute or two. Not sure what's going on there :/

File: build/updater.sh (hunk 2 above)
Comment (Ladsgroup) [reply build/updater.sh]: Done

File: build/updater.sh (hunk 1 above)
Comment (Ladsgroup) [reply build/updater.sh]: Done

**Review by jakobw — APPROVED** [review]
LGTM and haven't seen any further hiccups on the test system so yay!!

**Pull request merged**